lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -65,7 +89,16 @@ fn do_test_simple_monitor_temporary_update_fail(disconnect: bool) {
 	if disconnect {
 		nodes[0].node.peer_disconnected(&nodes[1].node.get_our_node_id(), false);
 		nodes[1].node.peer_disconnected(&nodes[0].node.get_our_node_id(), false);
-		reconnect_nodes(&nodes[0], &nodes[1], (true, true), (0, 0), (0, 0), (0, 0), (0, 0), (false, false));
```

Comment:
This also feels a bit awkward, no? Can we make it just put a line per *thing* if the total line is too long, not just all the time?

Reply:
I'm used to this look from previous formatters but fine with trying something different. 

>Can we make it just put a line per thing if the total line is too long, not just all the time? 

IIUC that is what it is: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#tall-default
We could change it to this instead: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#compressed

Reply:
Hmm, my question was more about when rustfmt decides to put it on new lines...I kinda assumed it was "too long" by char count, but looks like this one shouldn't hit that metric?

Reply:
I think the formatter fudges the character count a bit such that this line is barely over the limit: https://i.imgur.com/EWGbOBk.png 
We could increase the character count again from 120 to 150. that does seem a bit long to me but not a hill i'm willing to die on :stuck_out_tongue:

Reply:
Hmm...So my smallest terminal is 140-wide, though I presume I'm a minority here. Maybe bump again and see where we end up.

Reply:
Functions with large number of arguments may signal a need to refactor the code. Here, the arguments consisting of long sequences of literals aren't very meaningful to the reader.

Generally, I'd prefer if lines aren't longer than 100-120 characters. @valentinewallace Does `fn_args_layout` apply to function definition or calls (or both)?

Reply:
@jkczyz It appears to be both. I also prefer keeping line lengths to 100-120.

Reply:
@jkczyz looks like it applies to both. I also prefer 100-120 character max lines.